var/spack/repos/builtin/packages/hh-suite/package.py

```diff
@@ -19,15 +19,30 @@ class HhSuite(CMakePackage):
 
     version("3.3.0", sha256="dd67f7f3bf601e48c9c0bc4cf1fbe3b946f787a808bde765e9436a48d27b0964")
 
+    variant("shared", default=False, description="Build shared library")
     variant("mpi", default=True, description="Enable MPI support")
+    variant(
+        "simd",
+        default="AVX2",
+        description="SIMD instruction set",
+        values=("SSE2", "AVX2"),
+        multi=False,
+    )
 
     depends_on("cmake@2.8.12:", type="build")
     depends_on("mpi", when="+mpi")
 
-    def build_args(self, spec, prefix):
+    def cmake_args(self):
         args = []
+        if "+shared" in self.spec:
+            args.append(self.define_from_variant("BUILD_SHARED_LIBS", "shared"))
+            args.append(
+                self.define("CMAKE_EXE_LINKER_FLAGS", "-static -static-libgcc -static-libstdc++")
```

Comment:
Why this change? Please undo this, it doesn't work generally.

Reply:
I followed what they defined in their CI builds that's why:
https://github.com/soedinglab/hh-suite/blob/4c0cd66434ce0b83ccd247053f57989fdd53d82b/azure-pipelines.yml#L62C26-L62C26
But I will remove it and try again